lang/luasocket/patches/040-remove-fpic.patch

```diff
@@ -0,0 +1,13 @@
+--- a/src/makefile
++++ b/src/makefile
+@@ -164,8 +164,8 @@ DEF_linux=-DLUASOCKET_$(DEBUG) -DLUA_$(COMPAT)_MODULE \
+ 	-DUNIX_API='__attribute__((visibility("default")))' \
+ 	-DMIME_API='__attribute__((visibility("default")))'
+ CFLAGS_linux= -I$(LUAINC) $(DEF) -pedantic -Wall -Wshadow -Wextra \
+-	-Wimplicit -O2 -ggdb3 -fpic -fvisibility=hidden
+-LDFLAGS_linux=-O -shared -fpic -o 
++	-Wimplicit -O2 -ggdb3 -fvisibility=hidden
```

Comment:
At least ```-Wimplicit```, ```-O2``` and ```-ggdb3``` can be removed. We shouldn't override toolchain unless there's a very good reason to do so.
```-Wall```  already sets ```-Wimplicit``` https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html

Reply:
I like it. Size from 52272 to 50815 on ramips.